pkg/configmap/configmap.go

```diff
@@ -0,0 +1,66 @@
+package configmap
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	kubetypes "k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GcpProjectOperatorConfigMap store informations required for operations
+type GcpProjectOperatorConfigMap struct {
```

Comment:
@boranx This struct should reflect that data we are pulling from the configMap so in this case 

```
OperatorConfigMap struct {
BillingAccount string 'json:"billingaccount"`
ParentFolderID string 'json:"parentfolderid"
}
```

This way we can have a function the gets the configmap and unmarshalls it into this object and we can pass this object around our code. As we add stuff to the configmap all we need to do is add it to this object. 


so we would unmarshall `configmap.Data` later right into this struct

I also think we should change the variables to the following 

```
"parentFolderID" // Since now we are not longer using the ORG root ID ,but a different folder for each env
"billingAccount" // just formatting so we are consistent 
```

Reply:
we should also move 

`const orgGcpConfigMap = "gcp-project-operator"`

to the new configmap file and maybe `OperatorConfigMapName` would be better and the configmap data struct can be `OperatorConfigMap`